master/usage/configuration/ip-in-ip.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Configuring IP-in-IP
+---
+
+If your network fabric performs source/destination address checks 
+and drops traffic when those addresses are not recognized, it may be necessary to
+enable IP-in-IP encapsulation of the inter-workload traffic.
+  
+This is often the case for public-cloud environments where you have limited control
+over the network, and in particular you have no option to set up BGP peering between
+your Calico nodes and the network routers.
+
+Calico can be configured to use IP-in-IP encapsulation by enabling the IPIP option
+on the [IP pool resource]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/ippool). 
+When enabled, Calico will use IP-in-IP encapsulation when routing packets *to*
+workload IPs falling in the IP pool range.
+
+An optional `mode` field toggles when IP-in-IP is used, see following sections for
+details.
+   
+### Configuring IP-in-IP for all inter-workload traffic
+
+With the IP-in-IP `mode` set to `always`, Calico will route using IP-in-IP for
+all inter-workload traffic to workloads in the configured pool.
+
+The following `calicoctl` example will create or modify an IPv4 pool with 
+CIDR 192.168.0.0/16 to use IP-in-IP with mode `always`:
+
+```
+$ calicoctl apply -f - << EOF
+apiVersion: v1
+kind: ipPool
+metadata:
+  cidr: 192.168.0.0/16
+spec:
+  ipip:
+    enabled: true
+    mode: always
+EOF
+```
+
+> Note that the default value for `mode` is `always`, and therefore may be omitted
+> from the request, but is shown above for clarity.
+
+### Configuring cross-subnet IP-in-IP
+
+If IP-in-IP encapsulation is required for L3-hops, but not required within
+the L2 subnet, then set the IP-in-IP mode to `cross-subnet`.  With this mode, Calico
+will route using IP-in-IP encapsulation for inter-workload traffic to workloads in 
+the configured pool *only* if the  destination host is in a different subnet to the 
+originating host.
+
+This avoids unneccessary encapsulation within the same subnet.
+
+The following `calicoctl` example will create or modify an IPv4 pool with 
+CIDR 192.168.0.0/16 to use IP-in-IP with mode `cross-subnet`:
+
+
+```
+$ calicoctl apply -f - << EOF
+apiVersion: v1
+kind: ipPool
+metadata:
+  cidr: 192.168.0.0/16
+spec:
+  ipip:
```

Comment:
will do - I believe Eric was crafting some text regarding IPIP and NAT so unless you object I'll leave out the discussion why but I will include it in the YAML.